sites/funcampus/src/backend/funcampus/settings.py

```diff
@@ -539,6 +539,38 @@ class Development(Base):
         }
     }
 
+    RICHIE_DEMO_NB_OBJECTS = {"licences": 0}
```

Comment:
Your commit mention demo site as scope but you edit file in funcampus?

Reply:
True. It's the demo of funcampus. Changed it.